### PR TITLE
fix(movableCoord): fix options bug

### PR DESCRIPTION
**Pull request by sculove**
## Issue
#189 

## Details
- fix bug when options is null
- change qunt2.x style

## Preferred reviewers
@jongmoon

### CODE REVIEW

File: src/movableCoord.js [hunk 1]
```diff
@@ -667,22 +667,22 @@ MovableCoord에 대한 상세한 내용은 데모를 살펴보기 바란다.
 		},
 
 		// set up 'css' expression
-		_reviseOptions: function(options) {
+		_reviseOptions: function() {
 			var key;
+			var self = this;
 			$.each(["bounce", "margin", "circular"], function(i, v) {
```
Comment (kishu) [inline src/movableCoord.js]: You used `forEach` in `infiniteGrid`. [See it](https://github.com/naver/egjs/pull/330/files#diff-eaa190073eec9fe40432c1fa47c064f3R292)
Is there any reason can not use 'forEach`?

Comment (sculove) [inline src/movableCoord.js]: you're right. I suggest that we use es5 code. but, we don't decide it yet.
I refactor InfiniteGrid totally and I test it antecedently.

**Comment by kishu** [comment]
LGTM

**Comment by jongmoon** [comment]
LGTM!

**Pull request merged**